Automattic/jetpack
# Instant Search: Fix default overlay focus styles for better compatibility with some themes.

#### Changes proposed in this Pull Request:


Sets some default focus styles for buttons and checkboxes in the overlay and tweaks those as needed for specific sort options, secondary close, and the "Clear filters" button.

Also fixes the offset when focusing the "X" close button.

#### Other information:


#### Does this pull request change what data or activity we track or use?



No

#### Testing instructions:






1. On a site with a Jetpack Search plan, enable the Twenty Twenty One theme.
2. Open the Overlay by appending `?s=&category=uncategorized` to your site address.
3. Confirm the buttons look correct in both Light & Dark themes.
4. Toggle the light/dark themes from `/wp-admin/admin.php?page=jetpack-search-configure`.

https://user-images.githubusercontent.com/40267301/183593894-6616af60-c783-48be-976c-b4c96ccb28fe.mp4
